dm/templates/resource_policy/resource_policy.py.schema

```diff
@@ -0,0 +1,239 @@
+      snapshotProperties:
+        $ref: '#/schemas/ResourcePolicySnapshotSchedulePolicySnapshotProperties'
+        description: |
+          Properties with which snapshots are created such as labels,
+          encryption keys.
+    type: object
+  ResourcePolicySnapshotSchedulePolicyRetentionPolicy:
+    description: |
+      Policy for retention of scheduled snapshots.
+    id: '#ResourcePolicySnapshotSchedulePolicyRetentionPolicy'
+    properties:
+      maxRetentionDays:
+        description: |
+          Maximum age of the snapshot that is allowed to be kept.
+        format: int32
+        type: integer
+      onSourceDiskDelete:
+        description: |
+          Specifies the behavior to apply to scheduled snapshots
+          when the source disk is deleted.
+        enum:
+        - APPLY_RETENTION_POLICY
+        - KEEP_AUTO_SNAPSHOTS
+        - UNSPECIFIED_ON_SOURCE_DISK_DELETE
+        enumDescriptions:
+        - ''
+        - ''
+        - ''
+        type: string
+    type: object
+  ResourcePolicySnapshotSchedulePolicySchedule:
+    description: |
+      A schedule for disks where the schedueled operations are performed.
+    id: '#ResourcePolicySnapshotSchedulePolicySchedule'
+    properties:
+      dailySchedule:
+        $ref: '#/schemas/ResourcePolicyDailyCycle'
+      hourlySchedule:
+        $ref: '#/schemas/ResourcePolicyHourlyCycle'
+      weeklySchedule:
+        $ref: '#/schemas/ResourcePolicyWeeklyCycle'
+    type: object
+  ResourcePolicySnapshotSchedulePolicySnapshotProperties:
+    description: |
+      Specified snapshot properties for scheduled snapshots created
+      by this policy.
+    id: '#ResourcePolicySnapshotSchedulePolicySnapshotProperties'
+    properties:
+      guestFlush:
+        description: |
+          Indication to perform a ?guest aware? snapshot.
+        type: boolean
+      labels:
+        additionalProperties:
+          type: string
+        description: |
+          Labels to apply to scheduled snapshots. These can be later
+          modified by the setLabels method. Label values may be empty.
+        type: object
+      storageLocations:
+        description: |
+          Cloud Storage bucket storage location of the auto snapshot
+          (regional or multi-regional).
+        items:
+          type: string
+        type: array
+    type: object
+  ResourcePolicyWeeklyCycle:
+    description: |
+      Time window specified for weekly operations.
+    id: '#ResourcePolicyWeeklyCycle'
+    properties:
+      dayOfWeeks:
+        description: |
+          Up to 7 intervals/windows, one for each day of the week.
+        items:
+          $ref: '#/schemas/ResourcePolicyWeeklyCycleDayOfWeek'
+        type: array
+    type: object
+  ResourcePolicyWeeklyCycleDayOfWeek:
+    id: '#ResourcePolicyWeeklyCycleDayOfWeek'
+    properties:
+      day:
+        description: |
+          Allows to define schedule that runs specified day of the
+          week.
+        enum:
```

Comment:
Can you please order it from Monday-Sunday, infalid?

Reply:
Re-ordered. 'INVALID' option removed. I've copied it from the type description. Also it's presented in the API documentation. Have no idea how it can be used.

Reply:
It is reasonable to explain what each enum actually means, but since it is not documented in the API docs, we should not display empty descriptions here. Thanks for removing it.